frontend/src/metabase/query_builder/components/filters/modals/InlineOperatorSelector/InlineOperatorSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo, useCallback } from "react";
+import TippyPopoverWithTrigger from "metabase/components/PopoverWithTrigger/TippyPopoverWithTrigger";
+import Icon from "metabase/components/Icon";
+
+import {
+  InlineOperatorContainer,
+  FieldTitle,
+  OperatorDisplay,
+  OptionContainer,
+  Option,
+} from "./InlineOperatorSelector.styled";
+import { FilterOperatorName } from "metabase-types/types/Metadata";
+
+interface InlineOperatorSelectorProps {
+  fieldName: string;
+  value?: FilterOperatorName;
+  operators?: any[];
+  onChange?: (operatorName: string) => void;
+}
+
+export function InlineOperatorSelector({
+  fieldName,
+  value,
+  operators,
+  onChange,
+}: InlineOperatorSelectorProps) {
+  const operatorDisplayName =
+    operators?.find(o => o.name === value)?.verboseName ?? value;
+
+  const canChangeOperator = !!onChange && !!operators;
+
+  return (
+    <InlineOperatorContainer>
+      <FieldTitle>{fieldName} </FieldTitle>
+      {!canChangeOperator && !!operatorDisplayName && (
+        <OperatorDisplay>{operatorDisplayName}</OperatorDisplay>
+      )}
```

Comment:
if we don't have an operator, we can still display the column name

![Screen Shot 2022-07-08 at 11 13 22 AM](https://user-images.githubusercontent.com/30528226/178039414-29d19392-fdf3-49a4-90ff-e3e95cb7c843.png)